python/gps/algorithm/config.py

```diff
@@ -22,6 +22,8 @@
     'cost': None,  # A list of Cost objects for each condition.
     # Whether or not to sample with neural net policy (only for badmm/mdgps).
     'sample_on_policy': False,
+    # Number of clusters to use if clustering samples.
+    'num_clusters': 0,
```

Comment:
Set default to None rather than 0 to be more explicit? Also, say in the comment that this is used in RF-GPS (unless it can also be used for traj opt?)